sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,79 @@
+(2884100,16,2149.392,-6161.496,1.675816,'Scarlet Miner'),
+(2884100,17,2134.892,-6166.246,0.9258156,'Scarlet Miner'),
+(2884100,18,2130.892,-6173.246,4.175816,'Scarlet Miner'),
+(2884100,19,2125.892,-6180.746,10.17582,'Scarlet Miner'),
+(2884100,20,2123.392,-6184.746,13.92582,'Scarlet Miner'),
+(2884100,21,2115.892,-6196.246,13.67582,'Scarlet Miner'),
+(2884101,1,2376.838,-5906.152,108.6577,'Scarlet Miner'),
+(2884101,2,2340.088,-5903.402,102.6577,'Scarlet Miner'),
+(2884101,3,2306.838,-5904.902,90.65775,'Scarlet Miner'),
+(2884101,4,2292.838,-5917.746,82.40775,'Scarlet Miner'),
+(2884101,5,2285.642,-5917.652,64.15775,'Scarlet Miner'),
+(2884101,6,2272.088,-5968.402,49.90775,'Scarlet Miner'),
+(2884101,7,2248.338,-5978.402,36.90775,'Scarlet Miner'),
+(2884101,8,2234.838,-5996.902,25.90775,'Scarlet Miner'),
+(2884101,9,2223.588,-6015.152,11.65775,'Scarlet Miner'),
+(2884101,10,2216.088,-6035.902,7.407745,'Scarlet Miner'),
+(2884101,11,2195.088,-6059.402,6.407745,'Scarlet Miner'),
+(2884101,12,2184.838,-6087.152,3.157745,'Scarlet Miner'),
+(2884101,13,2180.588,-6149.402,1.657745,'Scarlet Miner'),
+(2884101,14,2177.588,-6149.402,1.907745,'Scarlet Miner'),
+(2884101,15,2188.838,-6170.402,1.157745,'Scarlet Miner'),
+(2884101,16,2213.088,-6165.402,1.157745,'Scarlet Miner'),
+(2884101,17,2241.838,-6160.402,1.907745,'Scarlet Miner'),
+(2884101,18,2269.088,-6163.402,2.157745,'Scarlet Miner'),
+(2884101,19,2269.588,-6168.902,2.407745,'Scarlet Miner'),
+(2884101,20,2270.088,-6174.652,5.907745,'Scarlet Miner'),
+(2884101,21,2270.588,-6180.902,9.907745,'Scarlet Miner'),
+(2884101,22,2271.088,-6186.152,14.15775,'Scarlet Miner'),
+(2884101,23,2273.838,-6196.402,13.65775,'Scarlet Miner');
+UPDATE `creature_template` SET `ScriptName`='',`AIName`='SmartAI' WHERE `entry`=28841;
+DELETE FROM `smart_scripts` WHERE `source_type`=0 AND `entryorguid`=28841;
+DELETE FROM `smart_scripts` WHERE `source_type`=9 AND `entryorguid` IN (2884100,2884101,2884102);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(28841,0,0,0,54,0,100,0,0,0,0,0,0,69,0,0,0,0,0,0,8,0,0,0,0,2386.673,-5900.587,108.5761,0,'Scarlet Miner - On just summoned - Move'),
+(28841,0,1,2,34,0,100,0,8,0,0,0,0,66,0,0,0,0,0,0,23,0,0,0,0,0,0,0,0,'Scarlet Miner - On movement inform - Set orientation'),
+(28841,0,2,3,61,0,100,0,8,0,0,0,0,1,0,0,0,0,0,0,1,0,0,0,0,0,0,0,0,'Scarlet Miner - Linked event - Say line 0'),
+(28841,0,3,0,61,0,100,0,0,0,0,0,0,87,2884100,2884101,0,0,0,0,1,0,0,0,0,0,0,0,0,'Scarlet Miner - Linked event - Random action list'),
```

Comment:
I checked sniffs and there's only one path for the miner, the second one is for the mine car as it doesn't follow, it uses  it's own wp

Reply:
I see that there are 2 possible paths for miner (also seen on videos: [right](https://www.youtube.com/watch?v=QtarXq1cICU)/[left](https://www.youtube.com/watch?v=h4aHk7H6XfA))

Reply:
I will try to find the second one

Reply:
```
ServerToClient: SMSG_ON_MONSTER_MOVE (0xAAE6) Length: 130 ConnIdx: 2 Time: 08/29/2011 21:05:58.188 Number: 78824
GUID: Full:  Type: Vehicle Entry: 28817 Low: 
Toggle AnimTierInTrans: false
Position: X: 2389.58 Y: -5901.18 Z: 109.0213
Move Ticks: 34079376
Spline Type: 0 (Normal)
Spline Flags: 100663296 (Unknown7, Unknown8)
Move Time: 67960
Waypoints: 21
Waypoint Endpoint: X: 2272.302 Y: -6192.553 Z: 13.21605
[1] Waypoint: X: 2376.691 Y: -5906.117 Z: 108.6187
[2] Waypoint: X: 2339.941 Y: -5903.367 Z: 102.6187
[3] Waypoint: X: 2306.691 Y: -5904.867 Z: 90.6187
[4] Waypoint: X: 2292.941 Y: -5917.617 Z: 82.3687
[5] Waypoint: X: 2285.941 Y: -5943.367 Z: 64.1187
[6] Waypoint: X: 2272.191 Y: -5968.367 Z: 49.86869
[7] Waypoint: X: 2248.191 Y: -5978.367 Z: 36.86869
[8] Waypoint: X: 2234.691 Y: -5996.867 Z: 25.86869
[9] Waypoint: X: 2223.441 Y: -6015.117 Z: 11.61869
[10] Waypoint: X: 2215.941 Y: -6035.867 Z: 7.368694
[11] Waypoint: X: 2195.191 Y: -6059.367 Z: 6.368694
[12] Waypoint: X: 2184.941 Y: -6087.117 Z: 3.118694
[13] Waypoint: X: 2180.441 Y: -6119.367 Z: 1.618694
[14] Waypoint: X: 2177.691 Y: -6149.617 Z: 1.868694
[15] Waypoint: X: 2188.691 Y: -6170.367 Z: 1.368694
[16] Waypoint: X: 2213.191 Y: -6165.617 Z: 1.118694
[17] Waypoint: X: 2241.691 Y: -6160.367 Z: 1.868694
[18] Waypoint: X: 2269.191 Y: -6163.367 Z: 2.118694
[19] Waypoint: X: 2271.191 Y: -6185.867 Z: 14.11869
[20] Waypoint: X: 2272.941 Y: -6196.367 Z: 13.61869

ServerToClient: SMSG_ON_MONSTER_MOVE (0xAAE6) Length: 142 ConnIdx: 2 Time: 08/29/2011 21:05:58.188 Number: 78816
GUID: Full:  Type: Creature Entry: 28841 Low: 
Toggle AnimTierInTrans: false
Position: X: 2386.673 Y: -5900.587 Z: 108.5761
Move Ticks: 34079373
Spline Type: 0 (Normal)
Spline Flags: 100663296 (Unknown7, Unknown8)
Move Time: 67896
Waypoints: 24
Waypoint Endpoint: X: 2273.003 Y: -6191.717 Z: 13.23941
[1] Waypoint: X: 2376.838 Y: -5906.152 Z: 108.6577
[2] Waypoint: X: 2340.088 Y: -5903.402 Z: 102.6577
[3] Waypoint: X: 2306.838 Y: -5904.902 Z: 90.65775
[4] Waypoint: X: 2292.838 Y: -5917.652 Z: 82.40775
[5] Waypoint: X: 2285.838 Y: -5943.402 Z: 64.15775
[6] Waypoint: X: 2272.088 Y: -5968.402 Z: 49.90775
[7] Waypoint: X: 2248.338 Y: -5978.402 Z: 36.90775
[8] Waypoint: X: 2234.838 Y: -5996.902 Z: 25.90775
[9] Waypoint: X: 2223.588 Y: -6015.152 Z: 11.65775
[10] Waypoint: X: 2216.088 Y: -6035.902 Z: 7.407745
[11] Waypoint: X: 2195.088 Y: -6059.402 Z: 6.407745
[12] Waypoint: X: 2184.838 Y: -6087.152 Z: 3.157745
[13] Waypoint: X: 2180.588 Y: -6119.402 Z: 1.657745
[14] Waypoint: X: 2177.588 Y: -6149.402 Z: 1.907745
[15] Waypoint: X: 2188.838 Y: -6170.402 Z: 1.157745
[16] Waypoint: X: 2213.088 Y: -6165.402 Z: 1.157745
[17] Waypoint: X: 2241.838 Y: -6160.402 Z: 1.907745
[18] Waypoint: X: 2269.088 Y: -6163.402 Z: 2.157745
[19] Waypoint: X: 2269.588 Y: -6168.902 Z: 2.407745
[20] Waypoint: X: 2270.088 Y: -6174.652 Z: 5.907745
[21] Waypoint: X: 2270.588 Y: -6180.902 Z: 9.907745
[22] Waypoint: X: 2271.088 Y: -6186.152 Z: 14.15775
[23] Waypoint: X: 2273.838 Y: -6196.402 Z: 13.65775
```

Reply:
Here the 2 paths
``` 
path 1

ServerToClient: SMSG_ON_MONSTER_MOVE (0xF1A4) Length: 130 ConnIdx: 0 Time: 03/08/2011 18:40:05.000 Number: 23691
GUID: Full: 0xF1507091001852C8 Type: Vehicle Entry: 28817 Low: 1594056
Toggle AnimTierInTrans: false
Position: X: 2389.58 Y: -5901.18 Z: 109.0213
Move Ticks: 32871034
Spline Type: 0 (Normal)
Spline Flags: 805306368 (UsePathSmoothing, Animation)
Move Time: 60414
Waypoints: 21
Waypoint Endpoint: X: 2115.671 Y: -6196.452 Z: 13.31215
[1] Waypoint: X: 2376.875 Y: -5906.316 Z: 108.6667
[2] Waypoint: X: 2340.125 Y: -5903.316 Z: 102.4167
[3] Waypoint: X: 2306.625 Y: -5904.816 Z: 90.66675
[4] Waypoint: X: 2292.625 Y: -5917.566 Z: 82.41675
[5] Waypoint: X: 2285.625 Y: -5943.316 Z: 64.16675
[6] Waypoint: X: 2271.875 Y: -5968.316 Z: 49.66675
[7] Waypoint: X: 2248.125 Y: -5978.566 Z: 36.91675
[8] Waypoint: X: 2234.625 Y: -5997.066 Z: 25.91675
[9] Waypoint: X: 2223.375 Y: -6015.066 Z: 11.66675
[10] Waypoint: X: 2216.125, -6036.066, 7.416748
[11] Waypoint: X: 2195.375, -6061.316, 6.166748
[12] Waypoint: X: 2184.375, -6092.066, 2.166748
[13] Waypoint: X: 2179.625, -6120.316, 1.666748
[14] Waypoint: X: 2175.375, -6143.066, 1.416748
[15] Waypoint: X: 2167.875, -6159.566, 1.916748
[16] Waypoint: X: 2149.375, -6161.316, 1.666748
[17] Waypoint: X: 2134.875, -6166.316, 0.916748
[18] Waypoint: X: 2130.875, -6173.316, 4.166748
[19] Waypoint: X: 2125.875, -6180.816, 10.16675
[20] Waypoint: X: 2123.375, -6184.816, 14.16675

ServerToClient: SMSG_ON_MONSTER_MOVE (0xF1A4) Length: 134 ConnIdx: 0 Time: 03/08/2011 18:40:05.000 Number: 23687
GUID: Full: 0xF13070A9001852CA Type: Creature Entry: 28841 Low: 1594058
Toggle AnimTierInTrans: false
Position: X: 2386.673 Y: -5900.587 Z: 108.5761
Move Ticks: 32871033
Spline Type: 0 (Normal)
Spline Flags: 805306368 (UsePathSmoothing, Animation)
Move Time: 60830
Waypoints: 22
Waypoint Endpoint: X: 2119.111 Y: -6191.905 Z: 13.27556
[1] Waypoint: X: 2376.892 Y: -5906.246 Z: 108.6758
[2] Waypoint: X: 2340.142 Y: -5903.496 Z: 102.4258
[3] Waypoint: X: 2306.392 Y: -5904.746 Z: 90.67581
[4] Waypoint: X: 2292.642 Y: -5917.746 Z: 82.42581
[5] Waypoint: X: 2285.642 Y: -5943.496 Z: 64.17581
[6] Waypoint: X: 2271.892 Y: -5968.496 Z: 49.67582
[7] Waypoint: X: 2248.142 Y: -5978.496 Z: 36.92582
[8] Waypoint: X: 2234.642 Y: -5996.996 Z: 25.92582
[9] Waypoint: X: 2223.392 Y: -6014.996 Z: 11.67582
[10] Waypoint: X: 2216.142 Y: -6035.996 Z: 7.425816
[11] Waypoint: X: 2195.392 Y: -6061.246 Z: 6.175816
[12] Waypoint: X: 2184.392 Y: -6091.996 Z: 2.175816
[13] Waypoint: X: 2179.642 Y: -6120.246 Z: 1.675816
[14] Waypoint: X: 2175.392 Y: -6142.996 Z: 1.425816
[15] Waypoint: X: 2167.892 Y: -6159.496 Z: 1.925816
[16] Waypoint: X: 2149.392 Y: -6161.496 Z: 1.675816
[17] Waypoint: X: 2134.892 Y: -6166.246 Z: 0.9258156
[18] Waypoint: X: 2130.892 Y: -6173.246 Z: 4.175816
[19] Waypoint: X: 2125.892 Y: -6180.746 Z: 10.17582
[20] Waypoint: X: 2123.392 Y: -6184.746 Z: 13.92582
[21] Waypoint: X: 2115.892 Y: -6196.246 Z: 13.67582


path 2:

ServerToClient: SMSG_ON_MONSTER_MOVE (0xF1A4) Length: 130 ConnIdx: 0 Time: 03/08/2011 18:40:20.000 Number: 24044
GUID: Full: 0xF1507091001852F1 Type: Vehicle Entry: 28817 Low: 1594097
Toggle AnimTierInTrans: false
Position: X: 2389.58 Y: -5901.18 Z: 109.0213
Move Ticks: 32871775
Spline Type: 0 (Normal)
Spline Flags: 805306368 (UsePathSmoothing, Animation)
Move Time: 67960
Waypoints: 21
Waypoint Endpoint: X: 2272.302 Y: -6192.553 Z: 13.21605
[1] Waypoint: X: 2376.691 Y: -5906.117 Z: 108.6187
[2] Waypoint: X: 2339.941 Y: -5903.367 Z: 102.6187
[3] Waypoint: X: 2306.691 Y: -5904.867 Z: 90.6187
[4] Waypoint: X: 2292.941 Y: -5917.617 Z: 82.3687
[5] Waypoint: X: 2285.941 Y: -5943.367 Z: 64.1187
[6] Waypoint: X: 2272.191 Y: -5968.367 Z: 49.86869
[7] Waypoint: X: 2248.191 Y: -5978.367 Z: 36.86869
[8] Waypoint: X: 2234.691 Y: -5996.867 Z: 25.86869
[9] Waypoint: X: 2223.441 Y: -6015.117 Z: 11.61869
[10] Waypoint: X: 2215.941 Y: -6035.867 Z: 7.368694
[11] Waypoint: X: 2195.191 Y: -6059.367 Z: 6.368694
[12] Waypoint: X: 2184.941 Y: -6087.117 Z: 3.118694
[13] Waypoint: X: 2180.441 Y: -6119.367 Z: 1.618694
[14] Waypoint: X: 2177.691 Y: -6149.617 Z: 1.868694
[15] Waypoint: X: 2188.691 Y: -6170.367 Z: 1.368694
[16] Waypoint: X: 2213.191 Y: -6165.617 Z: 1.118694
[17] Waypoint: X: 2241.691 Y: -6160.367 Z: 1.868694
[18] Waypoint: X: 2269.191 Y: -6163.367 Z: 2.118694
[19] Waypoint: X: 2271.191 Y: -6185.867 Z: 14.11869
[20] Waypoint: X: 2272.941 Y: -6196.367 Z: 13.61869

ServerToClient: SMSG_ON_MONSTER_MOVE (0xF1A4) Length: 142 ConnIdx: 0 Time: 03/08/2011 18:40:20.000 Number: 24033
GUID: Full: 0xF13070A9001852F7 Type: Creature Entry: 28841 Low: 1594103
Toggle AnimTierInTrans: false
Position: X: 2386.673 Y: -5900.587 Z: 108.5761
Move Ticks: 32871772
Spline Type: 0 (Normal)
Spline Flags: 805306368 (UsePathSmoothing, Animation)
Move Time: 67896
Waypoints: 24
Waypoint Endpoint: X: 2273.003 Y: -6191.717 Z: 13.23941
[1] Waypoint: X: 2376.838 Y: -5906.152 Z: 108.6577
[2] Waypoint: X: 2340.088 Y: -5903.402 Z: 102.6577
[3] Waypoint: X: 2306.838 Y: -5904.902 Z: 90.65775
[4] Waypoint: X: 2292.838 Y: -5917.652 Z: 82.40775
[5] Waypoint: X: 2285.838 Y: -5943.402 Z: 64.15775
[6] Waypoint: X: 2272.088 Y: -5968.402 Z: 49.90775
[7] Waypoint: X: 2248.338 Y: -5978.402 Z: 36.90775
[8] Waypoint: X: 2234.838 Y: -5996.902 Z: 25.90775
[9] Waypoint: X: 2223.588 Y: -6015.152 Z: 11.65775
[10] Waypoint: X: 2216.088 Y: -6035.902 Z: 7.407745
[11] Waypoint: X: 2195.088 Y: -6059.402 Z: 6.407745
[12] Waypoint: X: 2184.838 Y: -6087.152 Z: 3.157745
[13] Waypoint: X: 2180.588 Y: -6119.402 Z: 1.657745
[14] Waypoint: X: 2177.588 Y: -6149.402 Z: 1.907745
[15] Waypoint: X: 2188.838 Y: -6170.402 Z: 1.157745
[16] Waypoint: X: 2213.088 Y: -6165.402 Z: 1.157745
[17] Waypoint: X: 2241.838 Y: -6160.402 Z: 1.907745
[18] Waypoint: X: 2269.088 Y: -6163.402 Z: 2.157745
[19] Waypoint: X: 2269.588 Y: -6168.902 Z: 2.407745
[20] Waypoint: X: 2270.088 Y: -6174.652 Z: 5.907745
[21] Waypoint: X: 2270.588 Y: -6180.902 Z: 9.907745
[22] Waypoint: X: 2271.088 Y: -6186.152 Z: 14.15775
[23] Waypoint: X: 2273.838 Y: -6196.402 Z: 13.65775

Reply:
In such case as far as I understand WP must start at the same time. This leads to car being in front of miner because he does extra steps or because waypoints intersect. 
Speed of miner and car are the same so there should be some kind of delay in their waypoints, I guess?

Reply:
Yes it's  nightmare as the path will not be the same as in retail.
If there's not other solution keep the follow

Reply:
In such case I will leave it as is. At least for now.